docs/migrating_to_16.md

```diff
@@ -11,16 +11,51 @@ Key points:
    * If needed, the default service specific volume directories defined in [bento.env](../etc/bento.env) can be overriden in your 
      `local.env`.
 * The WES schema has been updated, so the run database must be cleared.
+* The reference service has been updated, so a SQL migration must be run.
```

Comment:
"an SQL" rather than "a SQL"

... sorry

Reply:
"sequel" or "ess-queue-ell"... that is a question

Reply:
ah, I guess I use the geriatric version